PowerEditor/src/Notepad_plus.cpp

```diff
@@ -4281,7 +4281,7 @@ void Notepad_plus::dropFiles(HDROP hdrop)
 		HWND hWin = ::ChildWindowFromPointEx(_pPublicInterface->getHSelf(), p, CWP_SKIPINVISIBLE);
 		if (!hWin) return;
 
-		if ((_subEditView.getHSelf() == hWin) || (_subDocTab.getHSelf() == hWin) || currentView() == SUB_VIEW)
+		if ((_subEditView.getHSelf() == hWin) || (_subDocTab.getHSelf() == hWin))

```

Comment:
It kind of revert https://github.com/notepad-plus-plus/notepad-plus-plus/commit/5f13c847191b38313d456fd69f575c5d84137ea0, which is supposed to fix #14354

Reply:
@donho 
You are right, my previous fix was not universal.